assets/src/setup/pages/technical-background/index.js

```diff
@@ -2,11 +2,28 @@
  * WordPress dependencies
```

Comment:
This screen to be covered in #4705. Changes to this file are just for testing this PR.